src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Colors {
+    red,
```

Comment:
```suggestion
    RED,
```
неймінг констант за конвенцією повинен бути капсовим